components/collective-page/CategoryHeader.js

```diff
@@ -99,6 +99,7 @@ const getCategoryData = (intl, collective, category) => {
         ),
       };
     case NAVBAR_CATEGORIES.CONTRIBUTE:
+      // TODO: Hide "Become a financial contributor." if can't contribute
```

Comment:
That will be for https://github.com/opencollective/opencollective-frontend/pull/7628